### PR TITLE
Add SECURITY.md

### PR DESCRIPTION
**Pull request by vicpatel**
Please take a look.

### CODE REVIEW

File: security.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+which includes all source code repositories managed through our GitHub organizations.
+Here is the list of [Corteza Repositories](https://github.com/orgs/cortezaproject/repositories)
+
+If you believe you have found a security vulnerability in any Corteza repository, please report it to us as described below.
+
+## Reporting Security Issues
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Instead, please send email to [security@cortezaproject.org](mailto:security@cortezaproject.org).
```
Comment (tjerman) [inline security.md]: Is this a valid email address?
Please validate.

File: security.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+If you prefer to submit via forum, please report them to [Corteza Forum](https://forum.cortezaproject.org/).
```
Comment (tjerman) [inline security.md]: I would exclude the forum as it is public also, plus I think it's easier for us to track it if its via email (for now)

File: security.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+Please include the requested information listed below (as much as you can provide)
+to help us better understand the nature and scope of the possible issue:
+
+* Type of issue
+* Full paths of source file(s) related to the manifestation of the issue
+* The location of the affected source code (tag/branch/commit or direct URL)
+* Any special configuration required to reproduce the issue
+* Step-by-step instructions to reproduce the issue
+* Proof-of-concept or exploit code (if possible)
+* Impact of the issue, including how an attacker might exploit the issue
```
Comment (tjerman) [inline security.md]: Have the first bullet start with a capital letter (leave as is), the rest should start with a lower-case letter (change).
All lines except for the last one should end with a comma (,) and the last with a period (.)

**Review by tjerman — CHANGES_REQUESTED** [review]
Please follow the one sentence per line rule.
Other then that it seems fine to me.

File: security.md (hunk 1 above)
Comment (darh) [reply security.md]: validated. :)

File: security.md (hunk 2 above)
Comment (vicpatel) [reply security.md]: Yeah sure.

Comment (zmija) [reply security.md]: @vicpatel we'll have to use info@cortezaproject.org here instead of security@

**Pull request merged**